src/ncp/ncp_base_mtd.cpp

```diff
@@ -2054,6 +2054,29 @@ template <> otError NcpBase::HandlePropertyRemove<SPINEL_PROP_IPV6_MULTICAST_ADD
     return error;
 }
 
+template <> otError NcpBase::HandlePropertyGet<SPINEL_PROP_IPV6_UNICAST_ADDRESS_TABLE>(void)
```

Comment:
The `SPINEL_PROP_IPV6_ADDRESS_TABLE ` supports `Get`, `Insert` and `Remove` operations. Does `SPINEL_PROP_IPV6_UNICAST_ADDRESS_TABLE ` also need to support all of these operations?

Reply:
Currently `Insert` or `Remove` is not used and there are no cases to do so. (Because at this stage all the unicast addresses on wpan are generated by the Thread network). Let's add them when needed.

Reply:
It seems the insert and remove handler are simple to implement. So I directly added it in this PR.